lightautoml/dataset/roles.py

```diff
@@ -105,15 +105,19 @@ def from_string(name: str, **kwargs: Any) -> "ColumnRole":
             kwargs = {**{"seasonality": (), "base_date": True}, **kwargs}
             return DatetimeRole(**kwargs)
 
+        if name in ["date"]:
+            kwargs = {**{"seasonality": (), "base_date": True}, **kwargs}
```

Comment:
remove redundant name.
What about "base_date" in line 104? I suggest to include it into this condition